src/pages/using-spark/components/input.mdx

```diff
@@ -310,6 +314,37 @@ consider using a Select Box instead.
   hasHTML
 />
 
+<a id="huge-radio-button" class="docs-b-Link--anchor"/>
+
+### Huge Radio Button
+[Back to Variants](#variants)
+
+The Huge Radio component is often used when the question is the
+core focus of the page in a step by step process.
+One an option is selected the next step in the flow
+is determined.
+
+#### Guidelines
+
+- Ensure list is in logical order for inputs (groups will stack).
+- Options are short and concise.
+- Recommend content to not exceed 40 characters.
+- The inputs become stacked in a grid. Stacking will
```

Comment:
We want to make sure to say something about the different layout options using the stack component. Maybe we should link to it?